package.json

```diff
@@ -90,7 +90,6 @@
     "babel-plugin-transform-imports": "2.0.0",
     "cross-fetch": "3.1.4",
     "enzyme": "3.11.0",
```

Comment:
It could be possible to remove `enzyme` completely, or? I tried to grep for tests using it, but there don't seem to be any.

If there are any that I missed, those could be replaced with tests written with the react testing library instead.

Reply:
There is just a setup for tests where it is used, but there are no tests in this repo, which is ehm...unfortunate and we should work on it, I know :D

Reply:
@gabipodolnikova All in time. :) If we do start to write tests here, we should probably start writing them with the react testing library, since enzyme has been deprecated. We can remove the enzyme setup stuff then.

Reply:
Removed! :) Thanks for the suggestion, deleting stuff is the best :D